src/scripts/limnoria_test.py

```diff
@@ -36,23 +36,28 @@
 import time
 import shutil
 import fnmatch
+from tempfile import TemporaryDirectory
 started = time.time()
 
 import supybot
 import logging
 import traceback
 
 # We need to do this before we import conf.
-if not os.path.exists('test-conf'):
-    os.mkdir('test-conf')
+main_temp_dir = TemporaryDirectory()
+print(f"Temporary directory path: {main_temp_dir.name}")
```

Comment:
Move this to the `.detach()` case so it's printed at the end (more visible), and only if the directory is kept.